core/Db.php

```diff
@@ -184,6 +184,10 @@ public static function createReaderDatabaseObject($dbConfig = null)
 
         $db = @Adapter::factory($dbConfig['adapter'], $dbConfig);
 
+        if (!empty($dbConfig['aurora_readonly_read_committed'])) {
```

Comment:
fyi we might still want to update `global.ini.php` here maybe @justinvelluppillai @diosmosis ?

Reply:
Do we want to document the setting?

Reply:
I reckon it be good. Even if it's just useful mostly for us but then other devs will know what it's about.

Reply:
I'll document and create a new pull request to add the global.ini.php update